go/kbfs/libkbfs/folder_block_manager.go

```diff
@@ -1213,15 +1150,17 @@ func (fbm *folderBlockManager) doReclamation(timer *time.Timer) (err error) {
 
 		// Add a new gcOp to show other clients that they don't need
 		// to explore this range again.
-		return fbm.finalizeReclamation(ctx, nil, nil, latestRev)
+		return fbm.finalizeReclamation(ctx, nil, nil, lastRev)
+	} else if shortened {
+		complete = false
```

Comment:
whoa was this a bug?

Reply:
It was, though it probably didn't have much practical effect besides slowing down QR a bit.